contrib/README.md

```diff
@@ -0,0 +1,45 @@
+# Community contributions
+
+The `contrib` directory contains community contributions
+that are not yet ready to be part of the poliastro API,
+but are nonetheless useful for Astrodynamics Python users.
+
+If you have a small script or set of scripts
+related to Astrodynamics and Orbital Mechanics
+but you feel they are "ugly", "not polished",
+or you are just not sure "if anybody will find it useful",
+we would be thrilled if you shared it with us here!
+
+If you want to upload your code here,
+please follow these steps:
+
+- If it's only one script,
+  or comment at the top
+  specifying your name, email,
+  what the script does,
+  what extra libraries are needed (if any),
+  and cite any scientific references related to your work.
+- If it's a series of scripts,
+  create a new directory with a `lower_case` name
```

Comment:
We might rewrite this as " create a lowercase named directory having a REDME.md file holding your name...".